docs/content/ProgressBar.mdx

```diff
@@ -20,12 +26,43 @@ If you'd like to use ProgressBar inline, pass the `inline` boolean prop & **be s
 </>
 ```
 
-## Component props
+## Props
+
+### ProgressBar
+
+<PropsTable>
+  <PropsTableRow name="progress" type="number" description="Used to set the size of the green bar" />
```

Comment:
Is this required?

Reply:
As in a required prop? No, it's optional. Defaults to 0 if it's not passed in.